contracts/ics/chan/src/entry.rs

```diff
@@ -0,0 +1,128 @@
+use crate::error::{Error, Result};
+use alloc::vec::Vec;
+use ckb_ics_axon::handler::{
+    handle_channel_open_ack_and_confirm, handle_channel_open_init_and_try, Client, IbcChannel,
+    IbcConnections,
+};
+use ckb_ics_axon::message::{
+    Envelope, MsgChannelOpenAck, MsgChannelOpenConfirm, MsgChannelOpenInit, MsgChannelOpenTry,
+    MsgType,
+};
+use ckb_standalone_types::prelude::Entity;
+#[cfg(feature = "debugging")]
+use ckb_std::ckb_types::prelude::*;
+use ckb_std::error::SysError;
+use ckb_std::{ckb_constants::Source, high_level as hl};
+use rlp::decode;
+use tiny_keccak::{Hasher as _, Keccak};
+
+pub fn main() -> Result<()> {
+    let envelope = load_envelope()?;
+    match envelope.msg_type {
+        MsgType::MsgChannelOpenInit
+        | MsgType::MsgChannelOpenTry
+        | MsgType::MsgChannelOpenAck
+        | MsgType::MsgChannelOpenConfirm => {}
+        _ => return Ok(()),
+    }
```

Comment:
the channel cell will also be placed into the packet transaction for updating its sequence status and in which the `envelope.msg_type` is none of these, so this logic here will cancel the transaction running I guess